### DISCUSSION TITLE
Stable release date

### DISCUSSION THREAD

**Discussion by eduard-gavrila** [orestbida/cookieconsent#597] [General]
Hi, I am working on a project which uses cookieconsent as part of a storage consent management system and we need to make a decision on what version of the package to use. 
At the moment the latest version is a release candidate, `3.0.0-rc.17`. 
I was wondering if there is any timeline for releasing a stable version?
Many thanks!

**Comment by orestbida** [comment]
Hi @eduard-gavrila, the latest rc.17 is already stable, and will (very likely) be announced as the first stable release by the end of the next week.

**Reply by msimkunas** [reply]
Should new adopters use v3 already? Is v2 going to be supported after v3 is released?

**Reply by mkalamarz** [reply]
@orestbida any news about the stable version?

**Reply by orestbida** [reply]
The playground was just [completed](https://playground.cookieconsent.orestbida.com/). Expect the official release by the end of the week.
